src/components/explorer/ExplorerIndividualContent.js

```diff
@@ -42,6 +42,7 @@ class ExplorerIndividualContent extends Component {
 
         this.state = {
             backUrl: null,
+            currentTab: null,
             selectedTab: "overview",
         };
     }
```

Comment:
Is this an unrelated fix? "tab" here seems like it refers to the top level tabs : Overview, Exporer, etc rather than the tabs in the search results table. (also, is `selectedTab` dead code? If not, what's the difference betweeen currentTab and selectedTab?)

Reply:
Sure! I removed the legacy code.